src/views/htmlcontent/src/html/app.html

```diff
@@ -16,6 +16,8 @@
                         showHeader="true"
                         [resized]="dataSet.resized"
                         (mousedown)="navigateToGrid(i)"
+                        [selectionModel]="selectionModel"
```

Comment:
No it's there for reference only. Talking with Anthony he found it useful to have it for HTML intellisense, but it was getting quite out of date (as you can see from my update). In the future we could remove it, but I'm trying to be consistent for now.